tests/plugins/emoji/manual/editablenullerror.html

```diff
@@ -0,0 +1,9 @@
+<textarea id="editor"></textarea>
+
+<script>
+	if ( !CKEDITOR.env.gecko || bender.tools.env.mobile ) {
+		bender.ignore();
+	}
```

Comment:
The fix was applied to all browsers while the test covers only FF :thinking: I know there were no issue in other browsers but testing them ensures there are no regressions. So if the code change affects all browsers the test should cover all browsers too.